### PR TITLE
Beautify skill grid

### PR DESCRIPTION
**Pull request by spacegaier**
A few visual tweaks:

1. Ensure that all skill boxes in a row have the same height (flex box approach that also works with smaller screen sizes).
2. Use correct hover icon for footer (pointer instead of grab).

**After:**
image

### CODE REVIEW

**Comment by spacegaier** [comment]
> 
> 
> @almakantara is also working on a visual update of the My Skills page in #189. Perhaps you can coordinate this effort?

Did not see that, since the PR title only mentions the chat interface...

**Comment by spacegaier** [comment]
@almakantara Now that #189 has been merged I update my PR accordingly.

**Comment by almakantara** [comment]
> @almakantara Now that #189 has been merged I update my PR accordingly.

ok, thanks

File: views/devices_list.pug [hunk 1]
```diff
@@ -45,8 +39,7 @@ block content
                 if dev.ownerTier !== 'cloud' && dev.ownerTier !== 'global'
                   p= _("This device is owned by your %s.").format(dev.ownerTier)
           div.panel-footer
-            div.row
-              if !dev.isTransient
+            div.row
```
Comment (gcampax) [inline views/devices_list.pug:42]: I think at some point you had a change to disable the delete button if the skill is transient (cannot be deleted), but that got lost in the merge.

**Review by gcampax — CHANGES_REQUESTED** [review]
One small issue

File: views/devices_list.pug (hunk 1 above)
Comment (spacegaier) [reply views/devices_list.pug:42]: I think you reviewed an old PR version. In the current one, that code line is no longer there.

Comment (gcampax) [reply views/devices_list.pug:42]: I reviewed out of the "files changed" tab, and I checked on the command-line, the change is still there. Bad merge?

Comment (spacegaier) [reply views/devices_list.pug:42]: I neither see it in "Files Changes" nor in my local repo. Weird...

Comment (gcampax) [reply views/devices_list.pug:42]: Maybe merge the master branch again? It should clear out what are actual changes in the end.

Comment (spacegaier) [reply views/devices_list.pug:42]: Okay, I just realized, we meant different things 🤦 
I thought you were still seeing the `dev.isTransient` part although it is no longer there. My mistake.

After the redesign of the skill page from #189, my feeling was that the disabled delete button is not really needed anymore. In the original PR version I had added it as a way to ensure that the footer is visually consistent (esp. in terms of height), but that is now achieved without the button.
Given that now a button without a label is used, adding a disabled button would not be helpful IMHO. In the old PR version it was a text button, so we could clearly distinguish between the "Forget" active button and the disabled "Permanent" button with the label.

What do you think?

Comment (gcampax) [reply views/devices_list.pug:42]: I agree with you on the design, but I think there is an issue with the implementation (not sure if intentional or not). With master, the delete button will be added only if the device is not transient (can be deleted). With the PR as it stands, the delete button will be added unconditionally, which doesn't work.

Comment (spacegaier) [reply views/devices_list.pug:42]: Good point. I think this issue already exists in master today independent of this PR, but I will push a fix now.

Comment (spacegaier) [reply views/devices_list.pug:42]: Good point. I will push a fix.

**Review by gcampax — APPROVED** [review]
Thank you! Looks good now.

**Pull request merged**